packages/amplify_api/example/integration_test/graphql_tests.dart

```diff
@@ -44,7 +44,7 @@ void main() {
       final req = GraphQLRequest<String>(
           document: graphQLDocument, variables: <String, String>{'id': id});
       final response = await Amplify.API.mutate(request: req).response;
-      Map data = jsonDecode(response.data!);
+      jsonDecode(response.data!);
```

Comment:
Prefer to delete or wrap with try catch so its intention is clear.

Reply:
done